x-pack/plugins/observability_solution/apm/public/components/app/service_overview/index.tsx

```diff
@@ -60,7 +60,7 @@ export function ServiceOverview() {
   const showApmOverview =
     isEntityCentricExperienceViewEnabled === false || hasApmSignal || !hasSignal;
 
-  if (isPending(serviceEntitySummaryStatus)) {
+  if (serviceEntitySummaryStatus === FETCH_STATUS.LOADING) {
```

Comment:
Why did you change it?

Reply:
Cypress test were failing.

When the new "New experience" is off . We don't fire the summary endpoint thus the status is `not_initiated`

Reply:
ah yeah, that makes sense. Thank you.